Type error when defining browser, devices, operatingSystems.

### Which package is this bug report for? If unsure which one to select, leave blank

@crawlee/playwright (PlaywrightCrawler)

### Issue description

Hello,
When I am defining parameters such as browser, devices, operatingSystems in PlaywrightCrawlerOptions, and than try to compile it using tsc I get the following error:
```console

> scraper@0.1.0 build /home/mble/Code/repos/pluget/services/scraper
> tsc

src/main.ts:48:17 - error TS2322: Type '{ name: "edge"; minVersion: number; }' is not assignable to type 'BrowserName'.

48                 {
                   ~
49                   name: "edge",
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
50                   minVersion: 96,
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
51                 },
   ~~~~~~~~~~~~~~~~~

src/main.ts:52:17 - error TS2322: Type '{ name: "chrome"; minVersion: number; }' is not assignable to type 'BrowserName'.

52                 {
                   ~
53                   name: "chrome",
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
54                   minVersion: 96,
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
55                 },
   ~~~~~~~~~~~~~~~~~

src/main.ts:56:17 - error TS2322: Type '{ name: "firefox"; minVersion: number; }' is not assignable to type 'BrowserName'.

56                 {
                   ~
57                   name: "firefox",
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
58                   minVersion: 94,
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
59                 },
   ~~~~~~~~~~~~~~~~~

src/main.ts:60:17 - error TS2322: Type '{ name: "safari"; minVersion: number; }' is not assignable to type 'BrowserName'.

60                 {
                   ~
61                   name: "safari",
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
62                   minVersion: 15,
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
63                 },
   ~~~~~~~~~~~~~~~~~

src/main.ts:65:25 - error TS2322: Type '"desktop"' is not assignable to type 'DeviceCategory'.

65               devices: ["desktop", "mobile"],
                           ~~~~~~~~~

src/main.ts:65:36 - error TS2322: Type '"mobile"' is not assignable to type 'DeviceCategory'.

65               devices: ["desktop", "mobile"],
                                      ~~~~~~~~

src/main.ts:66:34 - error TS2322: Type '"windows"' is not assignable to type 'OperatingSystemsName'.

66               operatingSystems: ["windows", "macos", "linux", "android", "ios"],
                                    ~~~~~~~~~

src/main.ts:66:45 - error TS2322: Type '"macos"' is not assignable to type 'OperatingSystemsName'.

66               operatingSystems: ["windows", "macos", "linux", "android", "ios"],
                                               ~~~~~~~

src/main.ts:66:54 - error TS2322: Type '"linux"' is not assignable to type 'OperatingSystemsName'.

66               operatingSystems: ["windows", "macos", "linux", "android", "ios"],
                                                        ~~~~~~~

src/main.ts:66:63 - error TS2322: Type '"android"' is not assignable to type 'OperatingSystemsName'.

66               operatingSystems: ["windows", "macos", "linux", "android", "ios"],
                                                                 ~~~~~~~~~

src/main.ts:66:74 - error TS2322: Type '"ios"' is not assignable to type 'OperatingSystemsName'.

66               operatingSystems: ["windows", "macos", "linux", "android", "ios"],
                                                                            ~~~~~


Found 11 errors in the same file, starting at: src/main.ts:48

 ELIFECYCLE  Command failed with exit code 2.
```

### Code sample

```TypeScript
package.json: https://github.com/pluget/services/blob/main/scraper/package.json
main.ts with PlaywrightCrawler initialised: https://github.com/pluget/services/blob/main/scraper/src/main.ts
```


### Package version

crawlee 3.2.2, playwright 1.31.1

### Node.js version

node v18.14.2, typescript 4.9.5

### Operating system

NixOS 23.05pre460365.3c5319ad3aa (Stoat) x86_64

### Apify platform

- [ ] Tick me if you encountered this issue on the Apify platform

### I have tested this on the `next` release

_No response_

### Other context

![image](https://user-images.githubusercontent.com/23295125/222987448-d84886a3-d16a-4495-a436-2827da2f8148.png)

--- a/docs/guides/avoid_blocking_playwright.ts
+++ b/docs/guides/avoid_blocking_playwright.ts
@@ -1,4 +1,5 @@
 import { PlaywrightCrawler } from 'crawlee';
+import { BrowserName, DeviceCategory, OperatingSystemsName } from '@crawlee/browser-pool';
 
 const crawler = new PlaywrightCrawler({
     browserPoolOptions: {
@@ -6,14 +7,14 @@ const crawler = new PlaywrightCrawler({
         fingerprintOptions: {
             fingerprintGeneratorOptions: {
                 browsers: [{
-                    name: 'edge',
+                    name: BrowserName.edge,
                     minVersion: 96,
                 }],
                 devices: [
-                    'desktop',
+                    DeviceCategory.desktop,
                 ],
                 operatingSystems: [
-                    'windows',
+                    OperatingSystemsName.windows,
                 ],
             },
         },

--- a/docs/guides/avoid_blocking_puppeteer.ts
+++ b/docs/guides/avoid_blocking_puppeteer.ts
@@ -1,4 +1,5 @@
 import { PuppeteerCrawler } from 'crawlee';
+import { BrowserName, DeviceCategory } from '@crawlee/browser-pool';
 
 const crawler = new PuppeteerCrawler({
     browserPoolOptions: {
@@ -6,11 +7,11 @@ const crawler = new PuppeteerCrawler({
         fingerprintOptions: {
             fingerprintGeneratorOptions: {
                 browsers: [
-                    'chrome',
-                    'firefox',
+                    BrowserName.chrome,
+                    BrowserName.firefox,
                 ],
                 devices: [
-                    'mobile',
+                    DeviceCategory.mobile,
                 ],
                 locales: [
                     'en-US',

--- a/website/versioned_docs/version-3.0/guides/avoid_blocking_playwright.ts
+++ b/website/versioned_docs/version-3.0/guides/avoid_blocking_playwright.ts
@@ -1,4 +1,5 @@
 import { PlaywrightCrawler } from 'crawlee';
+import { BrowserName, DeviceCategory, OperatingSystemsName } from '@crawlee/browser-pool';
 
 const crawler = new PlaywrightCrawler({
     browserPoolOptions: {
@@ -6,14 +7,14 @@ const crawler = new PlaywrightCrawler({
         fingerprintOptions: {
             fingerprintGeneratorOptions: {
                 browsers: [{
-                    name: 'edge',
+                    name: BrowserName.edge,
                     minVersion: 96,
                 }],
                 devices: [
-                    'desktop',
+                    DeviceCategory.desktop,
                 ],
                 operatingSystems: [
-                    'windows',
+                    OperatingSystemsName.windows,
                 ],
             },
         },

--- a/website/versioned_docs/version-3.0/guides/avoid_blocking_puppeteer.ts
+++ b/website/versioned_docs/version-3.0/guides/avoid_blocking_puppeteer.ts
@@ -1,4 +1,5 @@
 import { PuppeteerCrawler } from 'crawlee';
+import { BrowserName, DeviceCategory } from '@crawlee/browser-pool';
 
 const crawler = new PuppeteerCrawler({
     browserPoolOptions: {
@@ -6,11 +7,11 @@ const crawler = new PuppeteerCrawler({
         fingerprintOptions: {
             fingerprintGeneratorOptions: {
                 browsers: [
-                    'chrome',
-                    'firefox',
+                    BrowserName.chrome,
+                    BrowserName.firefox,
                 ],
                 devices: [
-                    'mobile',
+                    DeviceCategory.mobile,
                 ],
                 locales: [
                     'en-US',

--- a/website/versioned_docs/version-3.1/guides/avoid_blocking_playwright.ts
+++ b/website/versioned_docs/version-3.1/guides/avoid_blocking_playwright.ts
@@ -1,4 +1,5 @@
 import { PlaywrightCrawler } from 'crawlee';
+import { BrowserName, DeviceCategory, OperatingSystemsName } from '@crawlee/browser-pool';
 
 const crawler = new PlaywrightCrawler({
     browserPoolOptions: {
@@ -6,14 +7,14 @@ const crawler = new PlaywrightCrawler({
         fingerprintOptions: {
             fingerprintGeneratorOptions: {
                 browsers: [{
-                    name: 'edge',
+                    name: BrowserName.edge,
                     minVersion: 96,
                 }],
                 devices: [
-                    'desktop',
+                    DeviceCategory.desktop,
                 ],
                 operatingSystems: [
-                    'windows',
+                    OperatingSystemsName.windows,
                 ],
             },
         },

--- a/website/versioned_docs/version-3.1/guides/avoid_blocking_puppeteer.ts
+++ b/website/versioned_docs/version-3.1/guides/avoid_blocking_puppeteer.ts
@@ -1,4 +1,5 @@
 import { PuppeteerCrawler } from 'crawlee';
+import { BrowserName, DeviceCategory } from '@crawlee/browser-pool';
 
 const crawler = new PuppeteerCrawler({
     browserPoolOptions: {
@@ -6,11 +7,11 @@ const crawler = new PuppeteerCrawler({
         fingerprintOptions: {
             fingerprintGeneratorOptions: {
                 browsers: [
-                    'chrome',
-                    'firefox',
+                    BrowserName.chrome,
+                    BrowserName.firefox,
                 ],
                 devices: [
-                    'mobile',
+                    DeviceCategory.mobile,
                 ],
                 locales: [
                     'en-US',

--- a/website/versioned_docs/version-3.2/guides/avoid_blocking_playwright.ts
+++ b/website/versioned_docs/version-3.2/guides/avoid_blocking_playwright.ts
@@ -1,4 +1,5 @@
 import { PlaywrightCrawler } from 'crawlee';
+import { BrowserName, DeviceCategory, OperatingSystemsName } from '@crawlee/browser-pool';
 
 const crawler = new PlaywrightCrawler({
     browserPoolOptions: {
@@ -6,14 +7,14 @@ const crawler = new PlaywrightCrawler({
         fingerprintOptions: {
             fingerprintGeneratorOptions: {
                 browsers: [{
-                    name: 'edge',
+                    name: BrowserName.edge,
                     minVersion: 96,
                 }],
                 devices: [
-                    'desktop',
+                    DeviceCategory.desktop,
                 ],
                 operatingSystems: [
-                    'windows',
+                    OperatingSystemsName.windows,
                 ],
             },
         },

--- a/website/versioned_docs/version-3.2/guides/avoid_blocking_puppeteer.ts
+++ b/website/versioned_docs/version-3.2/guides/avoid_blocking_puppeteer.ts
@@ -1,4 +1,5 @@
 import { PuppeteerCrawler } from 'crawlee';
+import { BrowserName, DeviceCategory } from '@crawlee/browser-pool';
 
 const crawler = new PuppeteerCrawler({
     browserPoolOptions: {
@@ -6,11 +7,11 @@ const crawler = new PuppeteerCrawler({
         fingerprintOptions: {
             fingerprintGeneratorOptions: {
                 browsers: [
-                    'chrome',
-                    'firefox',
+                    BrowserName.chrome,
+                    BrowserName.firefox,
                 ],
                 devices: [
-                    'mobile',
+                    DeviceCategory.mobile,
                 ],
                 locales: [
                     'en-US',

--- a/website/versioned_docs/version-3.3/guides/avoid_blocking_playwright.ts
+++ b/website/versioned_docs/version-3.3/guides/avoid_blocking_playwright.ts
@@ -1,4 +1,5 @@
 import { PlaywrightCrawler } from 'crawlee';
+import { BrowserName, DeviceCategory, OperatingSystemsName } from '@crawlee/browser-pool';
 
 const crawler = new PlaywrightCrawler({
     browserPoolOptions: {
@@ -6,14 +7,14 @@ const crawler = new PlaywrightCrawler({
         fingerprintOptions: {
             fingerprintGeneratorOptions: {
                 browsers: [{
-                    name: 'edge',
+                    name: BrowserName.edge,
                     minVersion: 96,
                 }],
                 devices: [
-                    'desktop',
+                    DeviceCategory.desktop,
                 ],
                 operatingSystems: [
-                    'windows',
+                    OperatingSystemsName.windows,
                 ],
             },
         },

--- a/website/versioned_docs/version-3.3/guides/avoid_blocking_puppeteer.ts
+++ b/website/versioned_docs/version-3.3/guides/avoid_blocking_puppeteer.ts
@@ -1,4 +1,5 @@
 import { PuppeteerCrawler } from 'crawlee';
+import { BrowserName, DeviceCategory } from '@crawlee/browser-pool';
 
 const crawler = new PuppeteerCrawler({
     browserPoolOptions: {
@@ -6,11 +7,11 @@ const crawler = new PuppeteerCrawler({
         fingerprintOptions: {
             fingerprintGeneratorOptions: {
                 browsers: [
-                    'chrome',
-                    'firefox',
+                    BrowserName.chrome,
+                    BrowserName.firefox,
                 ],
                 devices: [
-                    'mobile',
+                    DeviceCategory.mobile,
                 ],
                 locales: [
                     'en-US',

